hyperv/tools/hypestv/Cargo.toml

```diff
@@ -16,7 +16,7 @@ mesh.workspace = true
 pal_async.workspace = true
 
 anyhow.workspace = true
-clap.workspace = true
+clap = { workspace = true, features = ["wrap_help"] }
```

Comment:
Should we have this feature on other cli tools?

Reply:
Probably. #395.